MCprep_addon/conf.py

```diff
@@ -123,6 +125,13 @@ def __init__(self):
 		# that no reading has occurred. If lib not found, will update to [].
 		# If ever changing the resource pack, should also reset to None.
 		self.material_sync_cache = []
+		
+		# i18n using Python's gettext module
+		self.en_i18 = gettext.translation("mcprep", self.languages_folder, fallback=True, languages=['en'])
+
+		# We've made this a part of the env class 
+		# since it's probably better in the long run
+		self._ = self.en_i18.gettext
```

Comment:
This feels very odd to me, what is this line accomplishing?

Reply:
It creates a functions called `_` in the env class. Generally `_` is used for localizing strings (in Python as well as C++) as it's pretty easy to type and recognizable by localization tools. However, I've put it in env since `_` is also syntax for a variable that's unused (like in a loop), and I'm not sure if that would be affected by making `_` global

Reply:
Do you have some reference or examples in third party packages using _ for localization? More for my own learning. Definitely a fan of following convention, though want to make sure it remains clear as well. But it does make sense to use such short functions for things like that, since it'll be all over.

Reply:
The Python docs for [`gettext`](https://docs.python.org/3/library/gettext.html) use `_`:
```py
import gettext
gettext.bindtextdomain('myapplication', '/path/to/my/language/directory')
gettext.textdomain('myapplication')
_ = gettext.gettext
# ...
print(_('This is a translatable string.'))
```
This is also standard in the GNU gextext library for C (which is what the Python gettext is based on), and is the default keyword `xgettext` looks for:
https://en.m.wikipedia.org/wiki/Gettext#Operation